app/src/main/java/org/schabi/newpipe/player/helper/PlayerHelper.java

```diff
@@ -391,6 +391,13 @@ public static boolean globalScreenOrientationLocked(final Context context) {
                 context.getContentResolver(), Settings.System.ACCELEROMETER_ROTATION, 0) == 0;
     }
 
+    @NonNull
+    public static Integer getProgressiveLoadIntervalBytes(@NonNull final Context context) {
+        return 1024 * Integer.parseInt(Objects.requireNonNull(getPreferences(context).getString(
```

Comment:
Shouldn't that be ``1000``?
https://en.wikipedia.org/wiki/Kilobyte

Reply:
Oh man, my life has been a lie =D
![image](https://user-images.githubusercontent.com/5570482/155032128-9abbd4e7-b60a-4910-8bdb-3bef383ce8c8.png)

That aside, kB is now KiB. Hope it won't be confusing to users.